.rubocop.yml

```diff
@@ -137,3 +137,6 @@ Style/TrailingCommaInHashLiteral:
 #Allow both ['a', 'b'], %w[a b] and %w(a b) style arrays
 Style/WordArray:
   Enabled: false
+
+Style/RedundantCondition:
+  Enabled: false
```

Comment:
I would not disable this cop just for one thing that is hard to rewrite. Instead I'd fix the particular case. You can write:
```ruby
type = params.fetch(:type) { name.end_with?('.in-addr.arpa', '.ip6.arpa') ? 'PTR' : 'A' }
```